app/utils/cu-profiling.ts

```diff
@@ -13,6 +20,49 @@ export type InstructionCUData = {
 
 const MIN_VALUE = 150;
 
+function getComputeBudgetInstructionName(
+    instruction: ParsedInstruction | PartiallyDecodedInstruction
+): string | undefined {
+    try {
+        if (!ComputeBudgetProgram.programId.equals(instruction.programId)) {
+            return undefined;
+        }
+
+        if (!('data' in instruction) || !instruction.data) {
+            return undefined;
+        }
+
+        const dataBuffer =
+            typeof instruction.data === 'string'
+                ? Buffer.from(bs58.decode(instruction.data))
+                : Buffer.from(instruction.data);
+
+        const type = identifyComputeBudgetInstruction(dataBuffer);
+
+        // Map instruction type to readable name
+        const typeMap: Record<ComputeBudgetInstruction, string> = {
+            [ComputeBudgetInstruction.RequestUnits]: 'requestUnits',
+            [ComputeBudgetInstruction.RequestHeapFrame]: 'requestHeapFrame',
+            [ComputeBudgetInstruction.SetComputeUnitLimit]: 'setComputeUnitLimit',
+            [ComputeBudgetInstruction.SetComputeUnitPrice]: 'setComputeUnitPrice',
+            [ComputeBudgetInstruction.SetLoadedAccountsDataSizeLimit]: 'setLoadedAccountsDataSizeLimit',
```

Comment:
For some reason at the current state CompputeBudget::instruction_name is not displayed. Coiuld you please check?

Reply:
Looks like we support transaction page and not the transaction inspector page